languages/en.json

```diff
@@ -284,10 +288,13 @@
     "builder.event.create.role.help": "Enter name of role for event is.",
     "builder.event.create.place.title": "Place of event",
     "builder.event.create.place.help": "Enter place of the event.",
+    "builder.event.create.place.example": ["School", "Wall Street", "Wenceslas Square"],

```

Comment:
You are not allowed to do this. We are not accepting arrays of string in translations. Do https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split (of comma ",") and then https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join of string: so "School,Wall Street,Wenceslas Square". Do it in all of arrays that you added to translations.